docs/use.md

```diff
@@ -0,0 +1,180 @@
+# Usage
+
+`cf create-service pubsub default foobar`
+
+`cf bind-service myapp foobar -c '{"role": "pubsub.admin"}'`
+
+Notes:
+
+`create-service` calls take the following optional custom parameters, all as strings.
```

Comment:
The stuff below is the optional custom parameters, right? If so maybe swap this line and the one after it so you can say the following and then have it be immediately following.